feature-utils/poly-look/test/react-components/cardList.test.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import { Card, CardList } from "../../src/react-components";
+import "@testing-library/jest-dom";
+import { HistoryStub } from "../utils/history-stub";
+import { INITIAL_HISTORY_STATE } from "../../src/constants/state";
+
+const testH1 = "TestH1";
+
+const singleBasicCard = (
+  <Card>
+    <h1>{testH1}</h1>
+  </Card>
+);
+
+//testing navigation
+let clickedCard;
+const history = new HistoryStub();
+const testRoute = "/testRoute";
+const testStateChange = { scrollingProgress: 200 };
+const testOnClick = () => {
+  clickedCard = true;
+};
+const testButtonText = "Details";
+const testRouteNavigation = {
+  history,
+  route: testRoute,
+  onClick: testOnClick,
+  stateChange: testStateChange,
+  buttonText: testButtonText,
+};
+
+const singleNavigationCard = (
+  <Card navigation={testRouteNavigation}>
+    <h1>{testH1}</h1>
+  </Card>
+);
+
+const cardWithNoButtonText = (
+  <Card navigation={{ ...testRouteNavigation, buttonText: null }}></Card>
+);
+
+const cardWithNoHistory = (
+  <Card
+    navigation={{ ...testRouteNavigation, history: null, onClick: null }}
+  ></Card>
+);
+
+const cardWithNoRoute = (
+  <Card
+    navigation={{ ...testRouteNavigation, route: null, onClick: null }}
+  ></Card>
+);
+
+const buttonErrorText = "Card: DetailsButton must have text";
```

Comment:
This is going to be brittle too, unless you import the constants from where it's defined.